corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -2,7 +2,7 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-{% block head %}{{ block.super }}
+{% block js %}{{ block.super }}
```

Comment:
there is a template tag in this block which made it uncompressible. overlooked because build error seemed to be from something related to our travis setup.